cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -33,13 +33,38 @@ def store_flow_cell_data_in_status_db(
             sequencer_name=parsed_flow_cell.machine_name,
             sequenced_at=parsed_flow_cell.run_date,
         )
+        flow_cell = add_samples_to_flow_cell_in_status_db(
+            flow_cell_name=flow_cell.name,
+            sample_internal_ids=get_sample_internal_ids_from_sample_sheet(
+                sample_sheet_path=parsed_flow_cell.sample_sheet_path,
+                flow_cell_sample_type=parsed_flow_cell.sample_type,
+            ),
+            store=store,
```

Comment:
Checking the store for the flow cell did not make sense there since it is the process of adding it. I changed it so it passes the flow cell now. 

We need the store the retrieve the samples to append to the flow cell.